web/src/components/layout/scaffold/Scaffold.vue

```diff
@@ -42,6 +46,8 @@ const props = withDefaults(defineProps<Props>(), {
   // eslint-disable-next-line vue/no-boolean-default
   enableTabs: false,
   activeTab: '',
+  // eslint-disable-next-line vue/no-boolean-default
+  noFluidWrapper: false,
```

Comment:
I think you can simply leave this out. The default is always false.

Reply:
Typescript is not able to infer that :(
This shows red squiggly line in vscode everywhere where it is expecting a boolean. (because it infers `boolean | undefined`)

Reply:
@dvjn `undefined` comes from the `?` marking `noFluidWrapper` as optional.

https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters

Reply:
Yes, but it needs to be optional, otherwise I'll have to pass it everywhere I create the component (or it'll give lint warnings there).

Reply:
What I originally meant to say is `defineProps` is not properly typed. In JS it converts all the `boolean | undefined` to `boolean` (by setting it to false) but doesn't reflect that in the returned prop's type unless you pass a default using `withDefaults`.

Reply:
@anbraten should I mark this resolved?

Reply:
@lukashass it it's resolved for you just resolve it :)